save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserPermissions.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.info
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property isPermittedCreateContest permission for create contests
+ * @property isPermittedToBulkUpload permission for upload COSV files
+ * @property organizationName name of organization
+ */
+@Serializable
+data class UserPermissions(
+    val isPermittedCreateContest: Boolean = false,
+    val isPermittedToBulkUpload: Boolean = false,
```

Comment:
user allowed to upload COSV and create contest only in organization. Organization should be not null

Reply:
Changed it